public/app/features/alerting/unified/Receivers.tsx

```diff
@@ -68,12 +106,33 @@ const Receivers: FC = () => {
 
   return (
     <AlertingPageWrapper pageId="receivers">
-      <AlertManagerPicker
-        current={alertManagerSourceName}
-        disabled={disableAmSelect}
-        onChange={setAlertManagerSourceName}
-        dataSources={alertManagers}
-      />
+      <div
+        className={css`
+          display: flex;
+          justify-content: space-between;
+        `}
+      >
+        <AlertManagerPicker
+          current={alertManagerSourceName}
+          disabled={disableAmSelect}
+          onChange={setAlertManagerSourceName}
+          dataSources={alertManagers}
+        />
+        {integrationsErrorCount > 0 && (
+          <NotificationError>
+            <div
+              className={css`
```

Comment:
Please, create a `getStyles` function for the styles in this component. The same as for the `NotificationError` component